src/frontend/note.js

```diff
@@ -104,6 +111,13 @@ class Note {
                 className: "fa fa-table",
                 title: "Insert Table"
             }, "|", {
+                name: "save",
+                action: () => { 
+                    notelist.save(this, this.#editor.value());
+                },
+                className: "fa fa-floppy-o no-disable",
```

Comment:
I `no-disable`correct? The save button has no effect in preview mode.

```suggestion
                className: "fa fa-floppy-o",
```